### ISSUE TITLE
Support IPv6

### ISSUE THREAD

**Issue by BoostCookie** [vladpen/cams#62]
If I insert my camera's IPv4 address then it works, but if I insert its IPv6 address then it does not work.

**Comment by vladpen** [comment]
The app don't restrict IPv6 access itself. Can you check IPv6 connection via command line VLC first?

Also, please note that when using IPv6, specifying the port (:554) in the camera address is required.

**Comment by BoostCookie** [comment]
Yes, you're right. VLC is the issue. If I put in the IPv6 of my cam into VLC (something like `rtsp://[fd00:1234::42]:554/live/...`) then vlc says

> VLC could not connect to "[fd00:1234".

which indicates that it has no idea how to handle the IPv6 address in the braces.

**Comment by vladpen** [comment]
Thank you.

**Comment by mooninite** [comment]
Upstream has tagged the fix for the 4.0 release, but there is no known release date for 4.0.  Is there a way you can compile a VLC with the fix or use the master branch of VLC to have this fix in Cams?

**Comment by mooninite** [comment]
4.X was due to be released two years ago. You will be waiting a few more years. Thanks for reviewing it though.

**Comment by vladpen** [comment]
Such a large delay is not accidental. VLC has blocking bugs that need to be fixed. It's pointless to patch their library. In this case, it is easier to consider using another engine.
